src/ovirtapi/transform.js

```diff
@@ -327,9 +327,9 @@ const VmStatistics = {
         stat.values.value &&
         (stat.name === 'disks.usage'
           ? stat.values.value[0] && stat.values.value[0].detail
-          : stat.values.value.length === 1
-            ? stat.values.value[0].datum
-            : stat.values.value.map(value => value.datum))
+          : (Array.isArray(stat.values.value)
+            ? stat.values.value.map(value => value.datum)
+            : [stat.values.value]))
```

Comment:
not sure but shouldn't it be just: 
[stat.values.value.datum]
instead of: 
[stat.values.value]

Reply:
// no values -> undefined, 1 value -> value.datum, >1 values -> [...values.datum]
      // ?disks.usage -> {detail...}
      // ?disks.usage -> {detail...}
Please update the comment on 323-324 as well